reana_server/fetcher.py

```diff
@@ -119,6 +119,7 @@ def fetch(self) -> None:
         if self._git_ref:
             repository.remote().fetch(self._git_ref, depth=1)
             repository.git.checkout(self._git_ref)
+        shutil.rmtree(os.path.join(self._output_dir, ".git"))
```

Comment:
For git repos we also download `.git` directory with lots of files which we have to move from one directory to another, so it's better to remove it here

Reply:
Is shallow copy still creating a lot of these internal files?

Anyway, it should be OK to remove top-level `.git` directory, as I guess users won't assume that we have access to `git` commands at the time of their workflow execution... Albeit could give some troubles if somebody would use `git lfs` or something for data access.  (We don't have such a use case yet I think.)

Reply:
For `roofit` example there are additional ~60 `git` files, additional 3 pages of files displayed in UI workspace tab. Perhaps later we could introduce some logic to keep these files in case workflow needs it